packages/app/auth/db.ts

```diff
@@ -73,6 +74,37 @@ class UsersDb {
             })
             .toArray()
     }
+
```

Comment:
These two new queries support NextAuth. Creating a user account on sign in and retrieving the mEditor user (to get at the user's roles)